frontend/style.less

```diff
@@ -0,0 +1,389 @@
+@color_1: #747474;
+@burgundy: #AE2222;
+@color_3: #535353;
+@color_4: #333;
+@color_5: #999;
+@color_6: #656565;
+@color_8: #AAAAAA;
+@color_9: #454545;
+@color_11: #FFA9A9;
+@color_12: #232323;
+@color_13: #A4A4A4;
+@helvetica: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
same with this, the name isn't very reusable here. maybe @default_font, @logo_font, and @label_font